en/getting-to-production.html

```diff
@@ -10,7 +10,10 @@
   zone as detailed in <a href="getting-started">getting started</a>.
 </p>
 
-<p>Prerequisites: git, Vespa CLI or openssl; for custom Java components: JDK 11 or higher, mvn 3.6 or higher.</p>
+<p>Prerequisites: git, Vespa CLI or openssl.</p>
```

Comment:
Make Vespa CLI a link to some appropriate page?

Reply:
Couldn't find one. Well, I guess there's the blog, but linking to that from doucmentation ... ?

Reply:
```suggestion
<p>Prerequisites: git, <a href="https://blog.vespa.ai/introducing-vespa-cli/">Vespa CLI</a> or openssl.</p>
```
link to Vespa CLI blog

Reply:
@mpolden, there should probably be an official page for the cli we can link to?

Reply:
https://docs.vespa.ai/en/vespa-cli.html